templates_apprunner/templates_cspm_eventbridge/FullInstall.yaml

```diff
@@ -0,0 +1,87 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: IAM Role for Agentless
+Parameters:
+  SysdigRoleName:
+    Type: String
+    Default: "SysdigAgentlessRole"
+    Description: Unique role for monitoring AWS accounts
+  SysdigExternalID:
+    Type: String
+    Description: Sysdig ExternalID required for the policy creation
+  SysdigTrustedIdentity:
+    Type: String
+    Description: Sysdig Trusted identity required for policy creation
+  SysdigEBName:
+    Type: String
+    Default: "sysdig"
+    Description: Unique name for role and policy
+  SysdigExternalID:
+    Type: String
+    Description: Sysdig ExternalID required for the policy creation
+  SysdigEventBusARN:
+    Type: String
+    Description: Sysdig event bus ARN
+  DeadLetterQueueARN:
```

Comment:
@cgeers do we have a plan with the FC ingestion team about DLQs?

Reply:
@cgeers and @nkraemer-sysdig we are removing DLQs from CFT and Terraform.